docs/pages/reference/access-controls/access-lists.mdx

```diff
@@ -46,14 +46,51 @@ and traits are granted by the Access List.
 
 ## Access List Membership
 
-Access List members are Teleport users who are granted the roles and traits specified
-by the Access List. Upon login, users will be granted these roles and traits along with
-their statically defined user permissions. These roles and traits will then tie into
-Teleport's existing RBAC system. Members can be optionally granted an expire date, after
-which their membership will no longer confer any grants to the user.
+Access List members are Teleport users or nested Access Lists who are granted the roles
+and traits specified by the Access List. Upon login, users will be granted these
+roles and traits along with their statically defined user permissions. These roles and
+traits will then tie into Teleport's existing RBAC system. Members can be optionally
+granted an expiry date, after which their membership will no longer confer any
+grants to the user.
 
 Members must meet requirements in order for their membership to be effective.
 
+## Nested Access Lists
```

Comment:
Feels like we should add information somewhere on how nested lists interact with Okta-synced lists somewhere. Maybe to this reference as well as adding a note somewhere in the guide?

Reply:
Yeah, could do!